roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -102,20 +106,12 @@ def local_images(self, images, task_vars):
 
     def is_image_local(self, image, task_vars):
         result = self.module_executor("docker_image_facts", {"name": image}, task_vars)
-        if result.get("failed", False):
```

Comment:
The fact that the module doesn't return `failed` explicitly today is an implementation detail. We should not rely on that on client code.
If for instance a module will raise an exception, let's say because of a missing dependency or programming error, Ansible captures it and would wrap the result with failed: True. It is useful to handle the case when module executions do fail.